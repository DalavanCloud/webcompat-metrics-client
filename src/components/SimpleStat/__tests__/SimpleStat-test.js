/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import { SimpleStat } from "../";

it("renders SimpleStat default correctly", () => {
  const tree = renderer
    .create(
      <SimpleStat>
        <div>{"simple stat"}</div>
      </SimpleStat>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders SimpleStat default correctly with custom style", () => {
  const tree = renderer
    .create(
      <SimpleStat style={{ color: "red" }}>
        <div>{"simple stat"}</div>
      </SimpleStat>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
