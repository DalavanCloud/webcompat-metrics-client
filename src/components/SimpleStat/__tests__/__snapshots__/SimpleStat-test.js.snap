// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SimpleStat default correctly 1`] = `
<div
  className="component"
>
  <div>
    simple stat
  </div>
</div>
`;

exports[`renders SimpleStat default correctly with custom style 1`] = `
<div
  className="component"
  style={
    Object {
      "color": "red",
    }
  }
>
  <div>
    simple stat
  </div>
</div>
`;
